sql-parser/src/main/java/io/crate/sql/tree/Intersect.java

```diff
@@ -56,8 +51,8 @@ public boolean isDistinct() {
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
-            .add("relations", relations)
-            .add("distinct", distinct)
+            .add("leftRelation", left)
```

Comment:
I would just use "left" and "right" for shorter names there, but up to you.

Reply:
Thanks, adjusted.